app/assets/javascripts/Components/Modals/filter_modal.jsx

```diff
@@ -0,0 +1,325 @@
+          return {key: item.name, display: item.name};
+        })
+      );
+    }
+    if (this.props.current_tags.length !== 0) {
+      options = options.concat(
+        this.props.current_tags.map(item => {
+          return {key: item.name, display: item.name};
+        })
+      );
+    }
+    return (
+      <MultiSelectDropdown
+        title={"Tags"}
+        options={options}
+        selected={this.props.filterData.tags}
+        onToggleOption={this.onToggleOptionTags}
+        onClearSelection={this.onClearSelectionTags}
+      />
+    );
+  };
+
+  rangeFilter = (min, max, title, onMinChange, onMaxChange) => {
+    return (
+      <div className={"filter"}>
+        <p>{title}</p>
+        <div className={"range"} data-testid={title}>
+          <input
+            className={"input-min"}
+            type="number"
+            step="any"
+            placeholder={"Min"}
+            value={min}
+            max={max}
+            onChange={e => onMinChange(e)}
+          />
+          <span>{I18n.t("to")}</span>
+          <input
+            className={"input-max"}
+            type="number"
+            step="any"
+            placeholder={"Max"}
+            value={max}
+            min={min}
+            onChange={e => onMaxChange(e)}
+          />
+          <div className={"hidden"}>
+            <FontAwesomeIcon icon={"fa-solid fa-circle-exclamation"} />
+            <span className={"validity"}>{I18n.t("results.filters.invalid_range")}</span>
+          </div>
+        </div>
+      </div>
+    );
+  };
+
+  onTotalMarkMinChange = e => {
+    this.props.mutateFilterData({
+      ...this.props.filterData,
+      totalMarkRange: {...this.props.filterData.totalMarkRange, min: e.target.value},
+    });
+  };
+
+  onTotalMarkMaxChange = e => {
+    this.props.mutateFilterData({
+      ...this.props.filterData,
+      totalMarkRange: {...this.props.filterData.totalMarkRange, max: e.target.value},
+    });
+  };
+
+  onTotalExtraMarkMinChange = e => {
+    this.props.mutateFilterData({
+      ...this.props.filterData,
+      totalExtraMarkRange: {...this.props.filterData.totalExtraMarkRange, min: e.target.value},
+    });
+  };
+
+  onTotalExtraMarkMaxChange = e => {
+    this.props.mutateFilterData({
+      ...this.props.filterData,
+      totalExtraMarkRange: {...this.props.filterData.totalExtraMarkRange, max: e.target.value},
+    });
+  };
+
+  componentDidMount() {
+    Modal.setAppElement("body");
+  }
+
+  onSubmit = event => {
+    event.preventDefault();
+    this.props.onRequestClose();
+  };
+
+  onClearFilters = event => {
+    event.preventDefault();
+    this.props.clearAllFilters();
+  };
+
+  render() {
+    if (this.props.loading) {
+      return "";
+    }
+    return (
+      <Modal
+        className="react-modal dialog"
+        isOpen={this.props.isOpen}
+        onRequestClose={() => {
+          this.props.onRequestClose();
+        }}
+      >
+        <h3>{I18n.t("results.filters.filter_by")}</h3>
```

Comment:
Remove this title text, and instead replace it with the same "Filter Submissions" text as the title of the filter button. Also put the filter icon beside it